crypto/rand/drbg_lib.c

```diff
@@ -265,7 +267,7 @@ size_t rand_drbg_get_nonce(RAND_DRBG *drbg,
         return 0;
 
     memset(&data, 0, sizeof(data));
-    pool = rand_pool_new(0, min_len, max_len);
+    pool = rand_pool_new(0, 0, min_len, max_len);
```

Comment:
Are nonces secret?  I'm not sure without more thought.  I'd err conservatively if there is any doubt.

@davidmakepeace @slontis any thoughts?

Reply:
Normally, nonces need not be secret, but must not repeat.

Reply:
Agree.